app/views/chefs/ingredients/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1>Ingredients Used by <%= @chef.name %></h1>
+<ul>
+  <% Chef.find(params[:chef_id]).ingredients.each do |ingredient| %>
+  <%= ingredient.name %>
+<% end %>
+</ul>
```

Comment:
You are doing AR in your view which goes against MVC. I mentioned earlier how you created an `@ingredients` variable but aren't using it.

```erb
<ul>
  <% @ingredients.each do |ingredient| %>
    <li><%= ingredient.name %></li>
  <% end %>
</ul>
```